### PR TITLE
internal/prometheus: add more buckets for job durations

### PR DESCRIPTION
**Pull request by jrusz**
We were hitting the limit on stage, let's increase it.

This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

File: internal/prometheus/job_metrics.go [hunk 1]
```diff
@@ -51,7 +51,7 @@ var (
 		Namespace: Namespace,
 		Subsystem: WorkerSubsystem,
 		Help:      "Duration a job spends on the queue.",
-		Buckets:   []float64{.1, .2, .5, 1, 2, 4, 8, 16, 32, 40, 48, 64, 96, 128, 160, 192, 224, 256, 320, 382, 448, 512, 640, 768, 896, 1024, 1280, 1536, 1792, 2049},
+		Buckets:   []float64{.1, .2, .5, 1, 2, 4, 8, 16, 32, 40, 48, 64, 96, 128, 160, 192, 224, 256, 320, 382, 448, 512, 640, 768, 896, 1024, 1280, 1536, 1792, 2048, 2304, 2560, 2816, 3072, 3328, 3584, 3840, 4096, 4352, 4608, 4864, 5120},
```
Comment (croissanne) [inline internal/prometheus/job_metrics.go:54]: I wonder if the last few should go by increments of 512 seconds? Now the limit is 80 minutes, would be nice maybe to be able to tell if it's 2 or 3 hours as well.

Comment (jrusz) [reply internal/prometheus/job_metrics.go:54]: Yeah sure that sounds better, I'll update it

Comment (jrusz) [reply internal/prometheus/job_metrics.go]: It's almost 3 hours but I hope it'll never come to that :sweat_smile:

Comment (croissanne) [reply internal/prometheus/job_metrics.go]: Perfect!

**Review by croissanne — APPROVED** [review]
thanks !

**Pull request merged**